ui/main.qml

```diff
@@ -69,7 +69,7 @@ StatusWindow {
 
     Action {
         shortcut: "Ctrl+W"
-        enabled: loader.item && !!loader.item.appLayout? loader.item.appLayout.appView.currentIndex === Constants.appViewStackIndex.browser
+        enabled: loader.item && !!loader.item.appLayout? loader.item.appLayout.appView.currentIndex === Constants.appSection.browser
```

Comment:
@alexandraB99 as mentioned above we cannot mix section types and indexes of a sections as component added to the `StackLayout`.